### PR TITLE
Upgraded admin-bundle to ~2.0@dev, changed master branch alias to 2.0-dev

### PR DESCRIPTION
**Pull request by chives**
I suggest to merge this first, then #32 and then all the rest PRs that depend from #32

### CODE REVIEW

**Comment by szymach** [comment]
:+1:

File: composer.json [hunk 1]
```diff
@@ -47,7 +47,8 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "1.1-dev"
+            "1.1": "1.1-dev",
```
Comment (norzechowicz) [inline composer.json]: no need for that, composer already create 1.1.x-dev from 1.1 branch.

File: composer.json [hunk 2]
```diff
@@ -12,7 +12,7 @@
     ],
     "require": {
         "php": ">=5.4.0",
-        "fsi/admin-bundle" : "~1.1@dev",
+        "fsi/admin-bundle" : "~2.0@dev",
```
Comment (norzechowicz) [inline composer.json]: ah also ``^`` would be better than ``~``, I missed that at first time

Comment (norzechowicz) [inline composer.json]: but it can be changed when admin will be stable.

**Pull request merged**